packages/boxed-expression-component/src/expressions/FunctionExpression/JavaFunctionExpression.tsx

```diff
@@ -285,15 +284,8 @@ function JavaFunctionExpressionLabelCell(props: React.PropsWithChildren<BeeTable
     return props.data[props.rowIndex].label;
   }, [props.data, props.rowIndex]);
 
-  useBeeTableSelectableCellRef(
-    props.rowIndex,
-    props.columnIndex,
-    undefined,
-    useCallback(() => label, [label])
-  );
-
   const getParameterLabelHelp = useCallback((): React.ReactNode => {
-    if (props.rowIndex === 1) {
+    if (props.rowIndex === 0) {
```

Comment:
Can you explain me this change? Because I'm not aware about how the rowSpan affects JavaFunctionExpression.

Reply:
yes, thank you, previously we merged a bug, please check hints for java function parameters before this PR and with this PR


### with this PR
![Screenshot 2023-03-31 080318](https://user-images.githubusercontent.com/8044780/229036394-64127b46-8e5d-4573-8dc5-8895cb453ef7.png)

### before this PR
the hint for `method signature` displayed hint for `class full qualified name` and vice versa